### DISCUSSION TITLE
Overlapping cell data

### DISCUSSION THREAD

**Discussion by Toby132** [satijalab/seurat#5137] [Q&A]
The seurat UMAP clustering algorithm clusters my cell data as multiple types within a single blob. As in there is 4 seurat clusters within one large space, and when I use my own labels, the clusters overlap.  Is it possible to get the large blob identity in seurat and then cluster that embedding  externally to do some sort of population testing/counting?

**Comment by saketkc** [comment]
The cluster identities and relevant metadata are in the `meta.data` slot:

```r
obj@meta.data
```

**Reply by Toby132** [reply]
Sorry, this gives me the each seurat cluster. What I am looking for is an identity for each individual island made by seurat.
